ArduCopter/AP_Arming.h

```diff
@@ -12,15 +12,15 @@ class AP_Arming_Copter : public AP_Arming
     {
         // default REQUIRE parameter to 1 (Copter does not have an
         // actual ARMING_REQUIRE parameter)
-        require.set_default(YES_MIN_PWM);
+        require.set_default((uint8_t)Required::YES_MIN_PWM);
     }
 
     /* Do not allow copies */
     AP_Arming_Copter(const AP_Arming_Copter &other) = delete;
     AP_Arming_Copter &operator=(const AP_Arming_Copter&) = delete;
 
     void update(void);
-    bool all_checks_passing(ArmingMethod method);
+    bool all_checks_passing(Method method);
```

Comment:
AP_Arming::Method ?

Reply:
We're within the correct scope.  We could add it for clarity only...